apps/web/vibes/soul/sections/product-detail/index.tsx

```diff
@@ -106,7 +106,7 @@ export function ProductDetail<F extends Field>({
 
                   <Stream
                     fallback={<ProductDetailFormSkeleton />}
-                    value={Promise.all([
+                    value={Streamable.all([
```

Comment:
Replacing `Promise.all` with `Streamable.all` in `ProductDetail` only. 

Other possible candidates are `ProductsList` and `ProductsListSection`, which we might want to change as well for consistency/to establish a pattern.

Reply:
Done.